apps/admin/pages/api/auth/[...nextauth].ts

```diff
@@ -1,18 +1,12 @@
-import NextAuth, { AuthOptions } from "next-auth";
+import NextAuth, { AuthOptions, Session } from "next-auth";
 import GoogleProvider from "next-auth/providers/google";
-import { PrismaAdapter } from "next-auth-prisma-adapter";
+import { Role } from "@prisma/client";
 import { prismaClient } from "db";
 import CredentialsProvider from "next-auth/providers/credentials";
 
 export const authOptions: AuthOptions = {
-  // debug: true,
+  debug: false,
   secret: process.env.NEXTAUTH_SECRET,
-  adapter: PrismaAdapter(prismaClient, {
```

Comment:
@mshd this is a good boilerplate for JWT session strategy, the adapter is actually not needed that much because:
- Session is not stored in DB anymore
- The user create/retrieve is done in callback, no black magic